packages/runtime/container-runtime/src/deltaManagerProxies.ts

```diff
@@ -194,3 +198,47 @@ export class DeltaManagerSummarizerProxy extends BaseDeltaManagerProxy {
 		this.isSummarizerClient = this.deltaManager.clientDetails.type === summarizerClientType;
 	}
 }
+
+export class DeltaManagerPendingOpsProxy extends BaseDeltaManagerProxy {
+	public get minimumSequenceNumber(): number {
+		const minPendingSeqNum = this.pendingStateManager.minimumPendingMessageSequenceNumber;
+		if (
+			minPendingSeqNum !== undefined &&
+			minPendingSeqNum < this.deltaManager.minimumSequenceNumber
```

Comment:
Can it happen that minPendingSeqNum > this.deltaManager.minimumSequenceNumber ?
I think not, and if it's the case - we should assert that.

Reply:
Definitely, adding a comment to explain a scenario for such an occurrence. Just submitting a regular op should do the trick.